File: server/src/game/conf/BetConfig.go
package conf

import (
	//"bufio"
	"encoding/json"
	"fmt"
	//"io"
	"io/ioutil"
	"os"
	//"strings"
)

type BetConfigElem struct {
	Id int
	GroupId int
	BetMoney int
	
}

type BetConfig0 struct {
	Dic map[int]*BetConfigElem
	Arr []*BetConfigElem
}

var (
	BetConfig BetConfig0
)

func loadBetConfig(path string) {
	file, err := os.Open(path)
	defer func() {
		if file != nil {
			file.Close()
		}
	}()
	if err != nil {
		panic(fmt.Sprint("load config error, path=", path))
	}

	BetConfig.Dic = make(map[int]*BetConfigElem)
	BetConfig.Arr = make([]*BetConfigElem, 0, 32)

	bt,err  :=  ioutil.ReadAll(file)
	json.Unmarshal(bt, &BetConfig.Arr)
	for _,conf := range BetConfig.Arr{
		BetConfig.Dic[conf.Id] = conf
	}
	
}

File: client/src/config/BetConfig.ts
class BetConfig {
    private static _dic:Dictionary<BetConfig> = new Dictionary<BetConfig>();
    private static _arr:Array<BetConfig> = new Array<BetConfig>();

	 public Id :number ;
     public GroupId :number ;
     public BetMoney :number ;
    

    
    public constructor(){

    }


    public static get Dic():Dictionary<BetConfig>{
        return  BetConfig._dic;
    }
    public static get Arr():Array<BetConfig>{
        return BetConfig._arr;
    }
    public static parse(json:any[]){

        for(var i = 0; i < json.length; i++){
            let config:BetConfig = <BetConfig>json[i];
            BetConfig._arr.push(config);
            BetConfig._dic.add(config.Id,config);
        }
    }
}